LEAF_Request_Portal/form.php

```diff
@@ -3170,21 +3170,27 @@ public function query($inQuery)
     public function getDisabledIndicatorList($disabled)
     {
         $vars = array(':disabled' => (int)$disabled);
-        $res = $this->db->prepared_query('SELECT * FROM indicators
-											LEFT JOIN categories USING (categoryID)
-						                    WHERE indicators.disabled >= :disabled
-						    					AND categories.disabled = 0
-						    				ORDER BY name', $vars);
+        $strSQL = "SELECT indicatorID, name, format, description, categories.categoryName, ".
+                    "indicators.disabled FROM indicators ".
+                    "LEFT JOIN categories USING (categoryID) ".
+                    "WHERE indicators.disabled >= :disabled ".
+                    "AND categories.disabled = 0 ".
+                    "ORDER BY name";
+
+        $res = $this->db->prepared_query($strSQL, $vars);
 
         $data = array();
         foreach ($res as $item)
         {
             $temp = array();
```

Comment:
Name too old-school.  Needs to be more descriptive.  Remember, those coming behind you may be the same type of people we replaced.